code/__DEFINES/dcs/signals/signals_mob/signals_mob_carbon.dm

```diff
@@ -173,3 +173,22 @@
 #define COMSIG_MUTATION_GAINED "mutation_gained"
 ///Called from on_losing(mob/living/carbon/human/owner)
 #define COMSIG_MUTATION_LOST "mutation_lost"
+
+/// Called from /datum/species/proc/harm() (unarmed attack chain is the Bad Place)
+/// Sent whenever you perform a harmful attack with a bodypart
+/// SEND_SIGNAL(target, signal, user, target, user.get_active_hand, damage_received, attacked_bodypart)
+#define COMSIG_CARBON_BODYPART_ATTACK_PERFORMED "bodypart_attack_performed"
+/// As above, but sent to the victim of such unprovoked violence
+/// SEND_SIGNAL(target, signal, user, target, user.get_active_hand, damage_received, attacked_bodypart)
+#define COMSIG_CARBON_BODYPART_ATTACK_VICTIM "bodypart_attack_victim"
+/// Ideally would be called anywhere unarmed damage is performed but it will be an entire project
+/// to sort out the unarmed attack chain
+/// Note the inclusion of how much damage is received
```

Comment:
Doc comments will be applied to whatever the following line is, if this relates to the above define then move them up.